app.sh

```diff
@@ -8,16 +8,16 @@ if [ "$SUBCOMMAND" = "producer" ]
 then
     if [ "$DEBUG_LEVEL" -eq 1]
     then
-        exec faust --debug --loglevel debug -A producer main
+        exec pipenv run faust --debug --loglevel debug -A producer main
```

Comment:
pipenv wont be available in the image, as the base image contains micropipenv.

Reply:
also the packages will be available in the image directly, so pipenv run might not be needed

Reply:
> also the packages will be available in the image directly, so pipenv run might not be needed

You are right, that was my thought, but then weird, because thoth-messaging is installed

Reply:
if i pull the image and run the code, i find different weird error in thoth.messaging itself.
```
  File "/opt/app-root/src/consumer.py", line 23, in <module>
    from thoth.messaging import MessageBase, UnresolvedPackageMessage
  File "/opt/app-root/lib/python3.6/site-packages/thoth/messaging/__init__.py", line 20, in <module>
    from .message_base import MessageBase
  File "/opt/app-root/lib/python3.6/site-packages/thoth/messaging/message_base.py", line 41, in <module>
    class MessageBase:
  File "/opt/app-root/lib/python3.6/site-packages/thoth/messaging/message_base.py", line 45, in MessageBase
    ssl_context = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH, cafile=KAFKA_CAFILE)
  File "/usr/lib64/python3.6/ssl.py", line 468, in create_default_context
    context.load_verify_locations(cafile, capath, cadata)
FileNotFoundError: [Errno 2] No such file or directory
```
its able to detect thoth.messaging

Reply:
I understood, one moment :)

Reply:
https://github.com/thoth-station/messaging/pull/144

Reply:
Thanks @harshad16 :100:

Reply:
>   File "/opt/app-root/src/consumer.py", line 23, in <module>
>     from thoth.messaging import MessageBase, UnresolvedPackageMessage
>   File "/opt/app-root/lib/python3.6/site-packages/thoth/messaging/__init__.py", line 20, in <module>
>     from .message_base import MessageBase
>   File "/opt/app-root/lib/python3.6/site-packages/thoth/messaging/message_base.py", line 41, in <module>
>     class MessageBase:
>   File "/opt/app-root/lib/python3.6/site-packages/thoth/messaging/message_base.py", line 45, in MessageBase
>     ssl_context = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH, cafile=KAFKA_CAFILE)
>   File "/usr/lib64/python3.6/ssl.py", line 468, in create_default_context
>     context.load_verify_locations(cafile, capath, cadata)
> FileNotFoundError: [Errno 2] No such file or directory

And the error here should be because Kafka certificate is missing actually if you run it locally :)